README.md

```diff
@@ -111,6 +106,30 @@ echo -e "admin\n${CONCOURSE_ATC_PASSWORD}" | \
    fly -t $FLY_TARGET login -k -c "https://${DEPLOY_ENV}-concourse.cf.paas.alphagov.co.uk"
 ```
 
+#### SSH to deployed Concourse and microbosh
+
+During the deploy process, a keypair is created in aws to create the instance
```

Comment:
It's still quite not right: 
>`bosh-init` needs this key to be able  a know

[Line 113](https://github.com/alphagov/paas-cf/pull/43/files#diff-04c6e90faac2675aa89e2176d2eec7d8R113)